README.md

```diff
@@ -0,0 +1,203 @@
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Recipe-app
+  gem install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  bin/rails test test/models/Recipe-app_test.rb
+```
+
+### Deployment
+
+You can deploy the Recipe-app application using: google cloud, Microsoft Azure, Netlify, github, render and other platform by giving credit on using our template and/or contribute via pull requests (read more in the contributing section).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Didy
+
+GitHub: [@Didy](https://www.github.com/diarisdiakite)
+<br> Twitter: [@diarisdiakite](https://www.twitter.com/diarisd)
+<br>LinkedIn: [@diarisdiakite](https://www.linkedin.com/in/diariatou-diakite-67ab80165/)
+<br><br>
+
+GitHub: [@Ali]()
+<br> Twitter: [@Ali]()
+<br>LinkedIn: [@dAli]()
+<br><br>
+
+
+[`Back to top`](#readme-top)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Dynamic list creating by the users using the interface**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/diarisdiakite/Recipe-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like the [Recipe-app]() project, you can support our work. Please, visit [diaridiakite personal page](https://diarisdiakite.github.io/my-portfolio/), follow me on my social media handles [@linkedin.com/diarisd](https://www.linkedin.com/in/diariatou-diakite-67ab80165/) and [@twitter.com/diarisd]().
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse community to inspire and encourage everyday programmers and aspiring programmers.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+Here are the most common questions about this project.
+
+### `Can we use the application as an organization`
+
+### `Answer`
+Yes! Make sure you give us credit of the application. 
+
+## `Where can we reach out to you ?`
+
+### `Answer`
+You can reach out to the organisation team by email [diarisdiakite@gmail.com](diarisdiakite@gmail.com) and [...]()
+
+[`Back to top`](#readme-top)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The MIT link is broken, The readme file is supposed to be professional and all links inside it is working. Please add a valid link.

![image](https://github.com/diarisdiakite/Recipe-app/assets/68796232/4777c883-d698-49a5-a4bb-b9f29bf3050e)